webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
```

Comment:
same as above. what's the purpose of this module?

Reply:
Removed CleanWebpackPlugin since we do not need it to clean build folders before building.